api/preprints/serializers.py

```diff
@@ -499,6 +500,7 @@ class PreprintCreateSerializer(PreprintSerializer):
     # Overrides PreprintSerializer to make id nullable, adds `create`
     # TODO: add better Docstrings
     id = IDField(source='_id', required=False, allow_null=True)
+    doi = ser.CharField(write_only=True, required=False)
```

Comment:
Note: use `manual_doi` as discussed; and wondering if we should add `allow_null=True` is better since I remember we ran into [issues](https://github.com/CenterForOpenScience/gravyvalet/pull/282/files) with FE when setting `required=False` only.